### PR TITLE
CacheBackend: Use a configurable setting to specify cache key

### PR DESCRIPTION
**Pull request by Flauschbaellchen**
Using a static class variable allows applications using django-health-check as a 3rd-party package to read the key's identifier used to read/write the cache entry.

This is in particular useful, if the application needs to know which keys are written from other packages and are not part of the application logic itself.

### CODE REVIEW

File: health_check/cache/backends.py
Comment (skarzi) [inline health_check/cache/backends.py]: Why not make it configurable via some Django setting e.g. `HEALTH_CHECK = {'CACHE_CHECK_KEY': 'my-cache-check-key'}`?

Comment (Flauschbaellchen) [reply health_check/cache/backends.py]: @skarzi well, your right. My requirement was to _read_ the key's name, but changing it to another custom one would might solve other problems, too.
Also, if setting the key's name within the Django settings would still allow the developer to access it, so my requirement would still be solved :+1: 

I've updated my PR.

Comment (Flauschbaellchen) [reply health_check/cache/backends.py]: I've choosen to use `HEALTHCHECK_CACHE_KEY` instead of extending `HEALTH_CHECK`.
It seems that `HEALTH_CHECK` is only used for `psutils` and other "single" settings exists (e.g. `HEALTHCHECK_CELERY_PING_TIMEOUT`.
I hope that was the right decision.

File: README.md
Comment (skarzi) [inline README.md:118]:
```suggestion
    HEALTHCHECK_CACHE_KEY = 'custom_healthcheck_key'
```

File: docs/settings.rst
Comment (skarzi) [inline docs/settings.rst:87]:
```suggestion
     - 'djangohealtcheck_test'
```

**Review by skarzi — COMMENTED** [review]
nice work 👍

File: README.md
Comment (Flauschbaellchen) [reply README.md:118]: @skarzi For both suggestions I was about to add those quotes around them, but as they were also left out from the other settings, I decided to leave them as they are.

Maybe this would be another refactoring PR after merging this one? In this way we could fix the other settings, too.

Comment (skarzi) [reply README.md:118]: ok, so then it makes sense to add these quotes in a separate PR 👍

Comment (Flauschbaellchen) [reply README.md:118]: @skarzi What are the next steps here? Do you resolve the discussions in case you're fine with my decisions or am I allowed to do that? In my opinion all threads can be resolved/closed.

**Comment by Flauschbaellchen** [comment]
@skarzi Any news on this? I've resolved the threads as no more work needs to be done.

**Comment by skarzi** [comment]
@Flauschbaellchen I am not a maintainer of this package, just a contributor. I think this PR could be merged (and thank you - I am running your changes from this PR on prod in some of my projects for ~1 month 👍).

@frankwiles can you take a look at this PR?

**Review by frankwiles — APPROVED** [review]
Looks great.  Thanks!

**Pull request merged**
